ivy/functional/frontends/numpy/creation_routines/from_existing_data.py

```diff
@@ -34,3 +34,8 @@ def copy(a, order="K", subok=False):
 @handle_numpy_dtype
 def frombuffer(buffer, dtype=float, count=-1, offset=0, *, like=None):
     return ivy.frombuffer(buffer)
+
+
+@to_ivy_arrays_and_back
+def choose(arr, choices, mode=None):
+    return ivy.choose(arr=arr, choices=choices, mode=mode)
```

Comment:
I don't think you should be passing all arguments as `kwargs` when calling `ivy.choose`. Perhaps this is causing the error [here](https://github.com/unifyai/ivy/actions/runs/6246782847/job/16958054722?pr=23335#step:3:231)